frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -908,6 +911,7 @@ private void prepareDetailView(RenderRequest request, RenderResponse response) t
 
                 addProjectBreadcrumb(request, response, project);
                 request.setAttribute(PROJECT_OBLIGATIONS, SW360Utils.getProjectObligations(project));
+                request.setAttribute(IS_USER_ADMIN,PermissionUtils.isUserAtLeast(UserGroup.ADMIN, user) ? "Yes" : "No");
```

Comment:
You should also allow `UserGroup.SW360_ADMIN` here, which is for SW360 the same as `ADMIN`. These roles only differ on the level of liferay.

Reply:
Thanks for the information.I have modified the configuration accordingly.